node/pkg/tss/broadcast_logic.go

```diff
@@ -14,10 +18,108 @@ import (
 // voterId is comprised from the id and key of the signer, should match the guardians (in GuardianStorage) id and key.
 type voterId string
 
+// SECURITY NOTE: this function sets a sessionID to a message. Used to ensure no equivocation.
+//
+// We don't add the content of the message to the uuid, instead we collect all data that can put this message in a context.
+// this is used by the reliable broadcast to check no two messages from the same sender will be used to update the full party
+// in the same round for the specific session of the protocol.
+type hasUUID interface {
+	getUUID(loadDistKey []byte) (uuid, error)
+}
+type parsedMsg interface {
+	hasUUID // most important feature.
+	wrapError(error) error
+	getTrackingID() *common.TrackingID // can be nil too.
+}
+
+type parsedProblem struct {
+	*tsscommv1.Problem
+	issuer *tsscommv1.PartyId
+}
+
+// Ensures the parsedProblem implements the ftCommand interface.
+func (p *parsedProblem) ftCmd() {}
+
+const parsedProblemDomain = "tssProblemDomainSeperator"
```

Comment:
There is a type for TSS messages, and we are using it :) 
I can hash it if I use var instead of cosnt, but I don't like it. 

I don't think it’s the worst if there are different lengths for the domain separator.